extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/config/CredentialProviderConfig.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.vault.runtime.config;
+
+import static io.quarkus.vault.CredentialProvider.PASSWORD_PROPERTY_NAME;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class CredentialProviderConfig {
+
+    /**
+     * Database credentials role, as defined by https://www.vaultproject.io/docs/secrets/databases/index.html
+     * <p>
+     * One of database-creds-role or kv-path needs to be defined. not both.
+     */
+    @ConfigItem
+    public String databaseCredsRole;
```

Comment:
As mentioned before, not a big fan of `Creds`.

Reply:
I followed vault's [terminology](https://www.vaultproject.io/docs/secrets/databases/index.html#usage): `vault read database/creds/my-role`

I renamed it into `database-credentials-role`